cmd/snap-update-ns/utils.go

```diff
@@ -34,7 +34,10 @@ import (
 
 // not available through syscall
 const (
-	umountNoFollow = 8
+	umountNoFollow   = 8
+	TMPFS_MAGIC      = 0x01021994
+	SQUASHFS_MAGIC   = 0x73717368
+	EXT4_SUPER_MAGIC = 0xef53
```

Comment:
This is not proper Go style.

Reply:
Should I rename them to some new invented names? Those are just C constants from a header.

Reply:
Updated to use go style.